ci-operator/config/openshift-kni/eco-ci-cd/openshift-kni-eco-ci-cd-main__cnf-core-4.16.yaml

```diff
@@ -0,0 +1,85 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: ci
+    namespace: openshift
+    tag: golang-1.20
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+  - as: eco-ci-cd
+    cluster: build05
+    cron: 00 13 * * 1
+    steps:
+      env:
+        VERSION: '4.16'
+        LOCK_LBL_HOST: "registry.hlxcl14.lab.eng.tlv2.redhat.com"
+        CLUSTER_NAME: "hlxcl14"
+        PULL_SPEC: "quay.io/openshift-release-dev/ocp-release:4.16.3-x86_64"
+        OCP_MAJOR_VERSION: 4
+        OCP_MINOR_VERSION: 16
+        BASE_DOMAIN: "lab.eng.tlv2.redhat.com"
+        PREDEPLOY_VALIDATE: "false"
+        BAREMETAL_NET_IPV6: "false"
+        BAREMETAL_NET_IPV4: "true"
+        PROVISIONING_NET_IPV6: "false"
+        PROVISIONING_NETWORK_STATE: "Disabled"
+        MANAGEMENT_INTERFACE_TYPE: "RedFish-VirtualMedia"
+        DISCONNECTED_INSTALL: "true"
+        WORKER_ROOT_DISK_HINT_VALUE: "/dev/sda"
+        BYPASS_OLM_SYNC: "true"
+        OLM_USE_NEXUS: "true"
+        OLM_CATALOG_SOURCE: "redhat-operators-brew"
+        OPERATORS_LIST: "ptp-operator, sriov-network-operator, metallb-operator, sriov-fec-operator, kubernetes-nmstate-operator"
+        OLM_ANSIBLE_EXTRA_ARGS: |
```

Comment:
Please look at my [PR](https://github.com/openshift/release/pull/51506/files#diff-9e86464a676bee25c2983925035378f05d109015b418a2387a68395d2b9ca493R36), this string should be uri encoded, since the columns and space are not processed correctly with Ansible.

Reply:
changed